go/runtime/host/sandbox/sandbox.go

```diff
@@ -564,6 +573,17 @@ func (r *sandboxedRuntime) manager() {
 
 				continue
 			}
+
+			// Ensure the command queue is empty to avoid processing any stale requests after the
+			// runtime restarts.
+		drainLoop:
+			for {
+				select {
+				case <-r.ctrlCh:
```

Comment:
I guess we have to return notify the caller that this was canceled, as otherwise the `Abort` requests will wait until the ctx is canceled.